### PR TITLE
Add rezensent to the list of features probot apps

### PR DESCRIPTION
**Pull request by KnisterPeter**
Please add my app to the list of features probot apps.

##### Checklist

- [x] If you're adding a listing for your app, be sure your app is in the `/_apps/` folder before opening this Pull Request.
- [x] All comments in frontmatter need to be removed.
- [x] Performs a useful action through the GitHub API that solves an existing problem for developers: Ease the code review in big projects
- [x] Is original: for example, it does something not already done by an existing Probot app
- [x] [Has tests](https://github.com/KnisterPeter/rezensent/tree/main/test)
- [x] [Has documentation](https://github.com/KnisterPeter/rezensent/blob/main/README.md)
- [x] [Is open source](https://github.com/KnisterPeter/rezensent/blob/main/LICENSE)
- [x] [Has a license](https://github.com/KnisterPeter/rezensent/blob/main/LICENSE)
- [x] [Has a code of conduct](https://github.com/KnisterPeter/rezensent/blob/main/CODE_OF_CONDUCT.md)
- [x] Has someone willing to at least minimally maintain them for the near future: KnisterPeter

-----
[View rendered _apps/rezensent.md](https://github.com/KnisterPeter/probot.github.io/blob/patch-1/_apps/rezensent.md)

### CODE REVIEW

**Comment by KnisterPeter** [comment]
@gr2m Can you help me with this? Why is it failing?
I think some of the currently listed apps are causing this, because they do not exist anymore.

File: _apps/rezensent.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  HEAD of it's base branch. And all remaining team pull requests are
+  update/recreated.
+
+- When there are no changes left, the managed pull request is closed as all
+  changes are integrated into it's base branch.
+
+### Process description
+
+<p align="center">
+  <img src="./docs/process.drawio.svg" alt="process" />
```
Comment (gr2m) [inline _apps/rezensent.md:60]: This needs to be a full URL, otherwise the image will not load

**Review by gr2m — CHANGES_REQUESTED** [review]
Great app! Other than the image URL this looks good 👍🏼

File: _apps/rezensent.md (hunk 1 above)
Comment (KnisterPeter) [reply _apps/rezensent.md]: Right!

**Review by gr2m — APPROVED** [review]

**Pull request merged**